.stylish-haskell.yaml

```diff
@@ -63,7 +63,7 @@ steps:
       #   >     (concat, foldl, foldr, head, init, last, length)
       #
       # Default: after_alias
-      list_align: with_module_name
+      list_align: after_alias
```

Comment:
I had issue with this as well. Thank you for fixing it.

Reply:
I actually copied that from your PR, so thank you.